plugins/git-plugin/src/main/groovy/org/rundeck/plugin/scm/git/GitExportPlugin.groovy

```diff
@@ -144,18 +146,28 @@ class GitExportPlugin extends BaseGitPlugin implements ScmExportPlugin {
     @Override
     void cleanup() {
         git?.close()
-        git?.getRepository()?.close()
     }
 
     @Override
     void totalClean(){
+        repo?.close()
         git?.getRepository()?.close()
+        git?.close()
         File base = new File(config.dir)
-        try {
-            FileUtils.delete(base, FileUtils.RECURSIVE)
-        } catch(IOException e){
-            logger.error("Failed to delete repo folder")
+
+        if(base.exists()){
+            try {
+                if (System.getProperty("os.name").toLowerCase().contains("windows")) {
```

Comment:
A test is needed to check for this line to be true at some point and validate that "windowCacheConfig.install()" is called 1 time and also, that is called 0 time if the condition is not met